src/main/webapp/jsp/instructorFeedbackCopyModal.jsp

```diff
@@ -10,9 +11,11 @@
       id="<%= Const.ParamsNames.COPIED_FEEDBACK_SESSION_NAME %>"
       type="text"
       name="<%= Const.ParamsNames.COPIED_FEEDBACK_SESSION_NAME %>"
+      maxlength="<%= FieldValidator.FEEDBACK_SESSION_NAME_MAX_LENGTH %>"
```

Comment:
As mentioned above, why not `maxLength`?

Reply:
this is the [attribute name](https://www.w3schools.com/tags/att_input_maxlength.asp)